src/services/goals.js

```diff
@@ -382,10 +382,11 @@ export async function goalsForGrants(grantIds) {
   });
 }
 
-async function removeActivityReportObjectivesFromReport(reportId) {
+async function removeActivityReportObjectivesFromReport(reportId, objectiveIdsToRemove) {
```

Comment:
Should we be adding a transaction to these types of operations?

Reply:
Transactions are now handled for us automagically. You were out when the PR went in, but we now use https://sequelize.org/docs/v6/other-topics/transactions/#automatically-pass-transactions-to-all-queries. Each request get it's own singular transaction.

Reply:
https://github.com/HHS/Head-Start-TTADP/pull/869

Reply:
Nice find, seems like a large change.